semcore/core/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [2.11.0] - 2023-11-29
+
+### Changed
+
+- Renamed type `RemoveFields` to `EfficientOmit` and moved it into `InternalTypings`.
```

Comment:
Do we need to text about internal changes?

Reply:
I don't see any problems because of that... For me - it is more readable than `Changed some inner types`. But I could change it if you insist